src/enums.h

```diff
@@ -114,6 +114,7 @@ enum art_effect_passive : int {
     AEP_SNAKES, // Summons friendly snakes when you're hit
     AEP_INVISIBLE, // Makes you invisible
     AEP_CLAIRVOYANCE, // See through walls
+    AEP_CLAIRVOYANCE_PLUS, // See through walls to a larger distance
```

Comment:
Note that adding anything to a hardcoded enum is highly likely to scramble existing saves that use this enum. At the very least old saves from 0.C stable, which are still supposed to be handled.

Reply:
Would placing it at the end of the list be enough?

Reply:
Probably, put it right before `NUM_AEPS`.